pandas/core/interchange/column.py

```diff
@@ -190,6 +190,10 @@ def describe_categorical(self):
 
     @property
     def describe_null(self):
+        if isinstance(self._col.dtype, BaseMaskedDtype):
+            column_null_dtype = ColumnNullType.USE_BYTEMASK
+            null_value = 1
```

Comment:
Curious why 1 here. Is this a throwaway value?

Reply:
1 = true for the uint8_t byte mask we would get back from NumPy in a BaseMaskedDtype

Reply:
`describe_null` returns a tuple:

https://github.com/pandas-dev/pandas/blob/4019d41816e6114d41731638fe027a8b0597db72/pandas/core/interchange/dataframe_protocol.py#L291-L301

In this case, we have a bytemask, where `1` indicates missing values